pkg/aggregate/aggregate.go

```diff
@@ -58,11 +59,51 @@ var aggrToString = map[AggrType]string{
 	aggrTypeStddev: "stddev", aggrTypeStdvar: "stdvar", aggrTypeAll: "*",
 }
 
+var aggrToSchemaField = map[string]config.SchemaField{
+	"count":  {Name: "count", Type: "array", Nullable: true, Items: "double"},
+	"sum":    {Name: "sum", Type: "array", Nullable: true, Items: "double"},
+	"sqr":    {Name: "sqr", Type: "array", Nullable: true, Items: "double"},
+	"max":    {Name: "max", Type: "array", Nullable: true, Items: "double"},
+	"min":    {Name: "min", Type: "array", Nullable: true, Items: "double"},
+	"last":   {Name: "last", Type: "array", Nullable: true, Items: "double"},
+	"avg":    {Name: "avg", Type: "array", Nullable: true, Items: "double"},
+	"rate":   {Name: "rate", Type: "array", Nullable: true, Items: "double"},
+	"stddev": {Name: "stddev", Type: "array", Nullable: true, Items: "double"},
+	"stdvar": {Name: "stdvar", Type: "array", Nullable: true, Items: "double"},
+}
+
+func SchemaFieldFromString(split []string, col string) ([]config.SchemaField, error) {
+	fieldList := make([]config.SchemaField, 0, len(split))
+	if len(split) != 0 && split[0] != "" {
+		for _, s := range split {
+			if strings.Compare(s, "*") == 0 {
+				fieldList = make([]config.SchemaField, 0, len(aggrToSchemaField))
+				for _, val := range aggrToSchemaField {
+					fieldList = append(fieldList, getAggrFullName(val, col))
+				}
+				return fieldList, nil
+			} else {
+				field, ok := aggrToSchemaField[s]
+				if !ok {
+					return fieldList, fmt.Errorf("Invalid aggragator type %s", s)
+				}
+				fieldList = append(fieldList, getAggrFullName(field, col))
+			}
+		}
+	}
+	return fieldList, nil
+}
+
+func getAggrFullName(field config.SchemaField, col string) config.SchemaField {
+	fullName := fmt.Sprintf("_%s_%s", col, field.Name)
```

Comment:
for empty `col` it will yield `__FieldName`

Reply:
col is never empty (illegal)